libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -333,6 +375,143 @@ async def _acall_with_config(
             await run_manager.on_chain_end(dumpd(output))
             return output
 
+    def _batch_with_config(
```

Comment:
why does this need t ova an instance method again?

Reply:
For subclasses to use when implementing batch, see RunnableRetry

Reply:
not sure i understand ,why can't batch_with_config just be a function not on a method that RunnableRtry uses?

Reply:
I'm not opposed to moving them, but not in this PR please :)